rust/hyperlane-base/src/types/checkpoint_syncer.rs

```diff
@@ -164,8 +163,9 @@ impl MultisigCheckpointSyncerConf {
         for (key, value) in self.checkpointsyncers.iter() {
             let gauge =
                 validator_checkpoint_index.with_label_values(&[origin, &key.to_lowercase()]);
+            // FIXME we can't use 20 byte wallet address here for sealevel (32 bytes)
```

Comment:
This is actually ok - I think Steven originally wrote this under impression this would need to change, but validators are still using the EVM 20 byte address